### ISSUE TITLE
Profile image broken in home and board views

### ISSUE THREAD

**Issue by alexmitrani** [online-go/online-go.com#2417]
**Describe the bug**
There should be a profile image in the top right of both home and board views. Recently it appears as broken. 

**To Reproduce**
1. Go to home or board view
2. See error

**Expected behavior**
There should be a profile image in the top right of both home and board views.

**Screenshots**

![Screenshot_20231110_223715_Firefox](https://github.com/online-go/online-go.com/assets/27974470/0fcd429a-7090-4b7b-973f-5d4d361195c6)
![Screenshot_20231110_223651_Firefox](https://github.com/online-go/online-go.com/assets/27974470/0c9bd40e-18f0-43a0-9825-5b08889e742b)
![Screenshot_20231110_223550_Chrome](https://github.com/online-go/online-go.com/assets/27974470/99cf29c2-f313-4b7e-aa4b-82118b5a15c9)

**Desktop (please complete the following information):**
![image](https://github.com/online-go/online-go.com/assets/27974470/ff7dd5ae-2a71-42d5-8ac0-429e28afc7bb)

 - OS: Windows 11 Home
 - Browser: Chrome
 - Version: 119.0.6045.107 (Build oficial) (64 bits)

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy A14 5G
 - OS: Android 13
 - The same problem happens with both Chrome and Firefox browsers: 
 - 
![Screenshot_20231110_225915_Chrome](https://github.com/online-go/online-go.com/assets/27974470/7d6257b6-0b02-4cc0-ac13-a36f75ebf358)

 - Firefox 119.0.1 (Build #2015982995), dcb5c11871+
GV: 119.0-20231019122658
AS: 119.0

**Additional context**
I noticed that this started happening today, yesterday it was not a problem.

**Comment by alexmitrani** [comment]
Maybe this is related to recent changes affecting profile images:  https://github.com/online-go/online-go.com/commit/fd6ce64cccf76732e834df851ce3006e577f6d95

**Comment by alexmitrani** [comment]
I tried changing my profile image to a downsized version that is 96x95 pixels but it still appears as broken. 

![2016-09-18_12-00-29.jpg](https://github.com/online-go/online-go.com/assets/27974470/225dc8c2-5809-4569-a5fb-97b1ee257024)

**Comment by benjaminpjones** [comment]
Does this only happen on mobile? FWIW, I'm unable to repro on my Samsung Galaxy.

If you can reproduce on a desktop/laptop, console logs (ctrl +shift + J) would help to understand the issue.

**Comment by alexmitrani** [comment]
> Does this only happen on mobile? FWIW, I'm unable to repro on my Samsung Galaxy.
> 
> If you can reproduce on a desktop/laptop, console logs (ctrl +shift + J) would help to understand the issue.

I tested on a Windows laptop and the problem also occurs on Chrome on that PC.  I noticed it also affects other people's profile images, which in some cases don't display either, for instance:

![image](https://github.com/online-go/online-go.com/assets/27974470/0e99f1d2-f7b3-4103-9ea5-1f44fd53c89a)

I attach the console log as requested.  Hope this helps.  
[online-go.com-1699726114854.log](https://github.com/online-go/online-go.com/files/13326176/online-go.com-1699726114854.log)
